python/nano/src/bigdl/nano/pytorch/inference/optimizer.py

```diff
@@ -167,6 +173,17 @@ def optimize(self, model: nn.Module,
 
         model.eval()  # change model to eval mode
 
+        input_sample = tuple(next(iter(training_data))[:-1])
+        st = time.perf_counter()
+        try:
+            with torch.no_grad():
+                model(*input_sample)
+        except Exception:
+            invalidInputError(False,
+                              "training_data is incompatible with your model input.")
+            exit(1)
```

Comment:
why we need to exit the process manually?

Reply:
will delete this line.